io_test.go

```diff
@@ -221,8 +340,13 @@ func TestOutput(t *testing.T) {
 			cfg.RegularGrid(ctmdata, pop, popIndices, mr, emis),
 		},
 		CleanupFuncs: []DomainManipulator{
-			Output(TestOutputFilename, false, "TotalPop deaths", "TotalPop",
-				"TotalPM25", "PM2.5 emissions", "Baseline TotalPM25", "WindSpeed"),
+			Output(TestOutputFilename, false, map[string]string{
+				"TotalPopDeaths":    "coxHazard(loglogRR(TotalPM25), TotalPop, MortalityRate)",
+				"TotalPop":          "TotalPop",
+				"TotalPM25":         "TotalPM25",
+				"PM25Emissions":     "PM25Emissions",
+				"BaselineTotalPM25": "BaselineTotalPM25",
```

Comment:
The checkOutputNames() function does two checks: (1) that no unsupported characters are used in the shapefile names and (2) that the output names do not exceed 10 characters